lib/Sema/TypeCheckSwitchStmt.cpp

```diff
@@ -121,25 +127,25 @@ namespace {
       // potentially push an enormous fixit on the user.
       static const size_t MAX_SPACE_SIZE = 128;
 
-      size_t computeSize(TypeChecker &TC,
+      size_t computeSize(TypeChecker &TC, const DeclContext *DC,
```

Comment:
It's awkward to plumb the DC (and TC) through everything here -- can't we make them instance variables?

Reply:
Sure, that makes sense. I'll save it for the next half, though, where I implement `unknown:`.